src/components/LeftSidebar/SearchBox/SearchBox.vue

```diff
@@ -26,6 +26,8 @@
 			:label="placeholderText"
 			:show-trailing-button="isSearching"
 			trailing-button-icon="close"
+			@blur="isFocused(false)"
+			@focus="isFocused(true)"
```

Comment:
Seems more readable to me to pass methods and use `@focus` and `@blur` in LeftSidebar:
```suggestion
			v-on:"$listeners"
```